### ISSUE TITLE
Ansroid Install Metrics incorrect.

### ISSUE THREAD

**Issue by tamkin04iut** [Microsoft/code-push#410]
In my dev machine, I released a new version of my app to `code-push`. Than app (running on dev device) was able to get the new version. When I entered the command `code-push deployment ls Skills2Go-Android`, the `Install Metrics` shows wrong, `Active: 0% (1 of 0)` and `Total: 1`.

Do I need to add additional code? I just added following code in app resume.
`CodePush.sync({
	updateDialog: true,
	installMode: CodePush.InstallMode.IMMEDIATE
});` 

![screen shot 2017-03-06 at 3 51 29 pm](https://cloud.githubusercontent.com/assets/5265362/23625993/cd394d72-0285-11e7-8783-a01fff6bd644.png)

**Comment by max-mironov** [comment]
Hello @tamkin04iut, this really looks odd. Can you please answer the following questions:
1) Are you able to reproduce this many times or this happens just once?
2) If it is reproducing every time are you able to repro this with our [sample app](https://github.com/Microsoft/react-native-code-push/tree/master/Examples/CodePushDemoApp)?
3) Is it possible that you have any problems with internet connection when running the app?

**Comment by tamkin04iut** [comment]
@max-mironov 

1. Yes, every time I release `code-push release-react Skills2Go-android android -d QC -m` to my QC  deployment and app running on dev device got the update. After downloading the new version `Install Metrics` shows `Active: 0% (1 of 0), Total: 1`, but after few mins it start showing `Active: 0% (0 of 0), Total:1`.
For example: v8, v9 `Install Metrics` was `Active: 0% (1 of 0), Total: 1`, now showing `Active: 0% (0 of 0), Total:1`. Latest release v10 shows `Active: 0% (1 of 0), Total: 1`.
![screen shot 2017-03-07 at 12 27 42 pm](https://cloud.githubusercontent.com/assets/5265362/23665818/1aa27166-0334-11e7-95dd-020324fe65bc.png)

2. https://github.com/tamkin04iut/code-push-test
3. No, I don't have any internet connection problem.

**Comment by max-mironov** [comment]
Hello @tamkin04iut, thank you so much for your response!

I've tried your example and it works well for me.
However I've found a way to reproduce your issue by doing the following steps:
1) Create a new depoyment and add a few releases (confirmed that everything is correct)
2) Clear deployment history for current release by using `code-push deployment clear <appName> <deploymentName>` command
3) After that go ahead and release a new update for my deployment
4) Updated the app on device and verified that now I've broken metrics for this deployment.

So after all it looks like clearing deployment history and then releasing updates will break metrics and it looks like a bug and we will address it soon.

Can you please confirm that your steps were similar  to the steps above?
As for your second question about reducing number of Active deployments - that seems to be expected [due to the following:](https://github.com/Microsoft/code-push/blob/master/cli/README.md#deployment-management) 

> The install metrics have the following meaning:
Active - The number of successful installs that are currently running this release (i.e. if the user opened your app, they would see/run this version). **This number will increase and decrease** as end-users upgrade to and away from this release, respectively.

So as far as your go to next release your number is reduced from previous one (e.g. v9) and incremented to the next (v10).

**Comment by tamkin04iut** [comment]
@max-mironov ,

Sorry for my late reply. 

I never ran `code-push deployment clear <appName> <deploymentName>` command.

I had just created a new deployment and pushed changes to that newly created deployment.

If possible I try to do reproduce both your and my case and let you know how it goes.

Thanks

**Comment by digaus** [comment]
I also have this bug... ran clear command and then released update to iOS. Now shows  `Active: 0% (1 of 0)`
Is there still being worked on?

**Comment by max-mironov** [comment]
@digaus unfortunately this issue is still active, we are aware of it and added this item to our backlog.

**Comment by digaus** [comment]
Found out a bit more:

Via CLI it shows two releases as it should but when fetching via the Management SDK I get a V5 release with active being -1 (looks like history did not get cleared correctly):
```
v1:
active: 0
downloaded:1
failed:0
installed:1

v2:
active1
downloaded:1
failed:0
installed:1

v5:
active:-1
downloaded:0
failed:0
installed:0
```

**Comment by max-mironov** [comment]
@digaus yep that seems to be the same story with this, clearing deployment history can introduce such unpleasant things. Thanks for sharing this!
